### PR TITLE
Added support for project management

### PR DESCRIPTION
**Pull request by runz0rd**
For #95

### CODE REVIEW

**Comment by tbarn** [comment]
Hey @runz0rd! Thanks for doing this. Would you mind adding some docs for it? Just one example of each would be great. Also, since Organization Keys are a bit new to this SDK, could you add that to where we talk about [keys](https://github.com/keenlabs/KeenClient-PHP#configuring-the-client)?

**Comment by tbarn** [comment]
I can get this reviewed in the next few days too.

**Comment by runz0rd** [comment]
Sure thing!

File: src/Client/Resources/keen-io-3_0.php [hunk 1]
```diff
@@ -40,22 +40,72 @@
...
+            'parameters'  => array(
+                'organizationId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'projectId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'organizationKey' => array(
```
Comment (baumatron) [inline src/Client/Resources/keen-io-3_0.php]: Would be cool to see this defined elsewhere and reused. Maybe it gets defined at the top of the file as `$organizationKeyValue => array(...);` or something like that?

File: src/Client/Resources/keen-io-3_0.php [hunk 2]
```diff
@@ -40,22 +40,72 @@
...
+                    'location' => 'json',
+                    'type'     => 'object',
+                ),
             ),
         ),
 
         'getProjects' => array(
-            'uri'         => 'projects',
+            'uri'         => 'organizations/{organizationId}/projects',
             'description' => 'Returns the projects accessible to the API user, as well as '
```
Comment (baumatron) [inline src/Client/Resources/keen-io-3_0.php]: Maybe this should be updated to `'Returns information about all projects for the given organization.'` to match https://keen.io/docs/api/#get-projects

File: src/Client/Resources/keen-io-3_0.php [hunk 3]
```diff
@@ -65,7 +115,7 @@
         ),
 
         'getProject' => array(
-            'uri'         => 'projects/{projectId}',
+            'uri'         => 'organizations/{organizationId}/projects/{projectId}',
             'description' => 'GET returns detailed information about the specific project, '
```
Comment (baumatron) [inline src/Client/Resources/keen-io-3_0.php]: Maybe update this to `'Returns information about a project.`' to match https://keen.io/docs/api/#get-project.

**Comment by runz0rd** [comment]
Valid points, the changes will be made.
Also tests, definitely.

**Comment by runz0rd** [comment]
Im guessing the build failed due to introducing 2 new keys in the unit tests that were necessary for the tests I added: ORGANIZATION_KEY and ORGANIZATION_ID

File: src/Client/Resources/keen-io-3_0.php [hunk 4]
```diff
@@ -1,5 +1,14 @@
 <?php
 
+$organizationKeyValue = array(
```
Comment (stof) [inline src/Client/Resources/keen-io-3_0.php]: There is a drawback here: the huge config array is now a dynamic array, meaning that OPCache cannot keep it in shared memory anymore.

**Pull request closed without merging**